lib/user/jwt/client.js

```diff
@@ -597,30 +607,30 @@ class Client {
     } = e
 
     if (statusCode) {
+      log(statusCode)
+
       if (statusCode > 400 && statusCode < 500) {
         // Data does not exist - ie. expired
-        this.throwRequestError(statusCode)
+        this.throwRequestError(statusCode, statusCode, response)
       } else {
         if (e.error) {
           // Handle errors which have an error object
           const message = e.error.name || e.error.code || 'EUNSPECIFIED'
-          this.throwRequestError(statusCode, message)
+          this.throwRequestError(statusCode, message, response)
         } else {
           // Handle errors which have no error object
           const message = e.code || 'ENOERROR'
-          this.throwRequestError(statusCode, message)
+          this.throwRequestError(statusCode, message, response)
         }
       }
     } else if (e.error) {
       // Handle errors which have an error object
       const message = e.error.name || e.error.code || 'EUNSPECIFIED'
-      const statusCode = getErrorStatusCode(message)
-      this.throwRequestError(statusCode, message)
+      this.throwRequestError(getErrorStatusCode(message), message, response)
```

Comment:
There's no point declaring this as a variable, we may as well just pass the return from the function into the next function, and pass the response object down, too